static/src/stylesheets/module/content/tones/_tone-special-report.scss

```diff
@@ -18,17 +18,15 @@
 .tonal--tone-special-report {
     // Interactives need help because their headers are not tonal
     &.content--interactive {
-        .commentcount i {
-            @extend %i-comment--tone-neutral2;
+        .content__headline {
+            font-weight: normal;
+            color: $neutral-1;
+        }
 
-            @if $svg-support {
-                .svg & {
-                    @extend %svg-i-comment--tone-neutral2;
-                }
-            }
+        .content__series-label__link {
+            color: $neutral-2;
         }
 
-        .tone-colour,
```

Comment:
Removing `.tone-colour` made sure that the comments showed up as grey rather than yellow. I find this more readable, but I can switch it back to yellow if needed.